dspace/bin/dspace

```diff
@@ -39,6 +39,8 @@ if [ "$JAVA_OPTS" = "" ]; then
   JAVA_OPTS="-Xmx256m -Dfile.encoding=UTF-8"
 fi
 
+export JAVA_OPTS="$JAVA_OPTS -Dlog4j2.configurationFile=$DSPACEDIR/config/log4j2-cli.xml"
```

Comment:
This change MUST also be copied to the `dspace.bat` file.  Otherwise, this will not be fixed in Windows environments